pkg/scaffold/operator.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scaffold
+
+import (
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/pkg/scaffold/input"
+)
+
+type Operator struct {
+	input.Input
+}
+
+func (s *Operator) GetInput() (input.Input, error) {
+	if s.Path == "" {
+		s.Path = filepath.Join(deployDir, operatorYamlFile)
+	}
+	s.TemplateBody = operatorTemplate
+	return s.Input, nil
+}
+
+const operatorTemplate = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.ProjectName}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: {{.ProjectName}}
+  template:
+    metadata:
+      labels:
+        name: {{.ProjectName}}
+    spec:
+      containers:
```

Comment:
There are Ansible template pipelines in [`master`](https://github.com/operator-framework/operator-sdk/pull/486/files#diff-e35d7b9b13729eab0761345f9ffbf202R459) that aren't here. Are they still required?